### ISSUE TITLE
Extra column / option to display date-times as MJD

### ISSUE THREAD

**Issue by timstaley** [transientskp/banana#66]
cf https://redmine.astron.nl/lofar_issuetracker/issues/6746

@gemmaanderson:  "The current format of the observation date and time on Banana can be hard to parse depending on the type of analysis required. Could the Modified Julian Date also be included on Banana and the output CSV file?"

**Comment by ajstewart** [comment]
I think this would be a useful addition, for example this would make it simple to compare with another external light curve either directly from banana or in the output csv file. The UTC date format and MJD would cover the two main time formats that astronomers use, so while it could be argued that you could simply download the csv and convert it offline, having MJD available straight off the bat would be handy.

I did the below using astropy.time conversion in convert.py

Images Overview:
![images-overview-mjd](https://cloud.githubusercontent.com/assets/3306161/8459894/2a023fde-2018-11e5-8327-7e1d0a33ea90.png)

Running Catalog Detail:
![runningcat-mjd](https://cloud.githubusercontent.com/assets/3306161/8459913/4a74d2cc-2018-11e5-8a91-409a95bd0e01.png)

It is also present on the image detail page. Also it is included in the output csv, e.g.:

>#id, taustart_ts (UTC), taustart_ts (MJD), tau_time, freq_central, f_int, f_int_err, f_peak, f_peak_err, extract_type, eta_nu, V_nu
10125, "2013-03-10T23:38:07.006950", 56361.9848033, 30.00305, 149000000.0, 1.02584115999, 0.131804461097, 1.0201962263, 0.0756785974716,0, 0.0, 0.0

@gemmaanderson does this cover what you were looking for?

Does anyone have any opinions that it should be presented in a different way?

**Comment by gijzelaerr** [comment]
We could also add pull down menu and let the user select the formatting for various things. We just have a limited amount of space and it is getting quite cluttered. Same with the positions sexagesimal. 

problem is that it is actually quite some work to add this functionality in javascript.
